internal/backends/postgresql/postgresql.go

```diff
@@ -90,7 +95,26 @@ func collectionsStats(ctx context.Context, p *pgxpool.Pool, dbName string, list
 	)
 
 	row := p.QueryRow(ctx, q, args...)
-	if err := row.Scan(&s.countDocuments, &s.sizeTables, &s.sizeIndexes); err != nil {
+	if err = row.Scan(&s.countDocuments, &s.sizeTables, &s.sizeFreeStorage, &s.sizeIndexes); err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	// The free index storage size is the size of free space map of index relation
+	// according to below documentations. However, it returns zero after inserting documents,
+	// then creating and deleting indexes.
+	// See https://www.postgresql.org/docs/current/functions-admin.html
+	// and https://www.postgresql.org/docs/current/storage-fsm.html.
+	q = fmt.Sprintf(`
+		SELECT
+			COALESCE(SUM(pg_relation_size(quote_ident(schemaname)|| '.' || quote_ident(indexname), 'fsm')), 0)
+		FROM pg_indexes
+		WHERE schemaname = $1 AND tablename IN (%s)
+		`,
+		strings.Join(placeholders, ", "),
+	)
+
+	row = p.QueryRow(ctx, q, args...)
+	if err := row.Scan(&s.sizeIndexFreeStorage); err != nil {
```

Comment:
See comment about this query, it _should_ fetch non zero value but under my local testing, I only got zero value.